### ISSUE TITLE
Update GloBI darwin core archive

### ISSUE THREAD

**Issue by KatjaSchulz** [jhpoelen/eol-globi-data#165]
It looks like both the snapshot and stable versions of the darwin core archive available under Accessing Species Interaction Data (https://github.com/jhpoelen/eol-globi-data/wiki#accessing-species-interaction-data) are quite old.  I just tried to pull some data about flower visitors from them, and there aren't any although the data were added to GloBI in June.  While EOL harvesting is still on hiatus, we would appreciate a fresh DwC-A dump to poke around in.

**Comment by jhpoelen** [comment]
According to my records, the most recent snapshots were created Tuesday 8 Sept 2015 (yesterday). The latest release was created over a year ago, on 23 July 2014. 

Could you please confirm that the snapshots are recent as they can be?

Meanwhile, I'll try and make an effort to create a more recent release and perhaps automate the process while I am at it. This has been on my list for a long time, so thank you for bringing it forward.

**Comment by jhammock** [comment]
Hmm... The records in question were added a couple of months ago and are available in the browser, eg: http://www.globalbioticinteractions.org/browse/index.html#interactionType=visitsFlowersOf&resultType=json

I think I've found them in the DarwinCore Archive (from the source, Sarah E Miller. 6/25/2015. Species associations manually extracted from Robertson, C. 1929. Flowers and insects: lists of visitors to four hundred and fifty-three flowers. Carlinville, IL, USA). In the browser their interaction type is visits flowers of. In the DarwinCore it's interacts with.

Were you waiting on the EOL terms, Jorrit? We have these now:
http://eol.org/schema/terms/VisitsFlowersOf
http://eol.org/schema/terms/FlowersVisitedBy

**Comment by jhpoelen** [comment]
@KatjaSchulz I've created a data release v0.4 to have a stable pointer to the datasets generated earlier this week (Sept 8, 2014). I've updated the wiki accordingly. 

@jhammock thanks for noticing we need to revisit our eol interaction type mapping that are used in the darwin core archives. Currently, newly introduced interactions (e.g. visitsFlowerOf) are mapping to the default interactoin: interactWith. I'll create a separate issue for that shortly.
